README.md

```diff
@@ -136,3 +136,30 @@ To test performance it's best to build the binary and run outside of docker:
 
 For reference here's the `markitpy-doc` docset (50k markdown files) currently takes `14s` vs `several minutes` compared to
 existing surveyed tools
+
+# Release Process
+
+This section outlines the process for releasing a new version of this project.
+
+## Versioning
+
+This project uses [Semantic Versioning](https://semver.org/) and its version is
+automatically determined by [release-drafter](https://github.com/release-drafter/release-drafter)
+based on the labels of the pull requests merged into the `main` branch.
+
+See the [release-drafter configuration](../.github/release-drafter.yml) for more details.
+
+## Creating a New Release
+
+Every time a pull request is merged into the `main` branch, release-drafter will
+create a draft release or update the existing draft release in the [Releases](https://github.com/elastic/docs-builder/releases) page.
+
+To create a new release you need to publish the existing draft release created by release-drafter.
+
+> [!IMPORTANT]
+> Make sure the [release-drafter workflow](../.github/workflows/release-drafter.yml) is finished before publishing the release.
+
+> [!NOTE]
+> When a release is published, the [create-major-tag workflow](../.github/workflows/create-major-tag.yml)
+> will force push a new major tag in the format `vX` where `X` is the major version of the release.
+> For example, if the release is `v1.2.3` was published, the workflow will force push a new tag `v1` on the same commit.
```

Comment:
Can we drop the `v*` prefix from the release-drafter? Just because we are already set up with `Major.Minor.Patch` tags. 

Alternatively I can update our `minver` and workflow triggers.

Reply:
I expected this discussion.

We can use a version with the `v` prefix.

Would you also like to remove the automation for `v1`,`v2`, etc tags?

Reply:
https://github.com/elastic/docs-builder/pull/202/commits/0fc2d20109df4694d10e0596275f8c67b5f5f40e

Reply:
Thanks! yeah release tags being `\d.\d.\d` and special `v\d` tags for github actions works for me!